### PR TITLE
trivial: Reword guides to guide

### PR DESCRIPTION
**Pull request by sidt4**
Reword guides to guide.

### CODE REVIEW

**Comment by ximion** [comment]
This is incorrect though, as these links point to multiple guides, specifically to the landing page that has guides for addon maintainers, gui app maintainers, distributors, translators, ...

So, I don't think this is a correct change.

**Comment by sidt4** [comment]
It's not clear as it's the same link in https://github.com/ximion/appstream/commit/060991c22f739cd416726c3a0e72357556e796d4 ?

**Comment by ximion** [comment]
> It's not clear as it's the same link in [060991c](https://github.com/ximion/appstream/commit/060991c22f739cd416726c3a0e72357556e796d4) ?

Yeah, I guess my comment to change that was actually wrong :sweat_smile: 
It should have been "guides" there as well, I apparently misread that comment.

**Comment by sidt4** [comment]
Okay. Updated.

File: compose/asc-hint-tags.c [hunk 1]
```diff
@@ -59,7 +59,7 @@ AscHintTagStatic asc_hint_tag_list[] =  {
 	{ "ancient-metadata",
 	  AS_ISSUE_SEVERITY_WARNING,
 	  "The AppStream metadata should be updated to follow a more recent version of the specification.<br/>"
-	  "Please consult <a href=\"https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html\">MetaInfo quickstart guide</a> for "
+	  "Please consult <a href=\"https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html\">MetaInfo quickstart guides</a> for "
```
Comment (ximion) [inline compose/asc-hint-tags.c:62]:
```suggestion
	  "Please consult the <a href=\"https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html\">MetaInfo quickstart guides</a> for "
```

File: README.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ You can [download signed release tarballs of AppStream on freedesktop.org](https
 ## Information For Users
 
 ### For Upstream Application Authors
-- **[Quickstart Guides](https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html)**:
+- **[MetaInfo Quickstart Guides](https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html)**:
```
Comment (ximion) [inline README.md:37]: The guides have nothing to do with just MetaInfo data - one is explicitly about how to deploy AppStream data if you are a distributor, the other is about translations, and only two are about MetaInfo files. So I think the previous sentence in that context is good enough.

**Review by ximion — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (sidt4) [reply README.md:37]: But the heading in the page says `Metadata Quickstart`. Should that be updated too ?

appstream

Comment (ximion) [reply README.md:37]: I don't think so... All of this is concerning metadata...

**Review by ximion — APPROVED** [review]

**Pull request merged**